packages/ui-library/src/components/radio/index.test.ts

```diff
@@ -54,6 +55,25 @@ describe('blr-radio', () => {
     expect(errorClassName).to.contain('error');
   });
 
+  it('error Icon is set to undefined', async () => {
+    const element = await fixture(
+      BlrRadioRenderFunction({
+        ...sampleParams,
+        hasError: true,
+        errorIcon: undefined,
+      })
+    );
+
+    const labelWrapper = querySelectorDeep('.label-wrapper', element.getRootNode() as HTMLElement);
+    const captionWrapper = querySelectorDeep('.caption-wraper', labelWrapper?.getRootNode() as HTMLElement);
+    const formCaptions = querySelectorAllDeep('.blr-form-caption', captionWrapper?.getRootNode() as HTMLElement);
+    const formCaptionError = querySelectorDeep('.blr-form-caption', formCaptions[1] as HTMLElement);
+
+    const errorIcon = formCaptionError?.getAttribute('errorIcon');
```

Comment:
Please adjust this like other test because this will ends up the line 74 always being truthy.